Octokit.Tests/Models/IssueTest.cs

```diff
@@ -257,7 +257,7 @@ public void CreatesAnIssueUpdateRequestObject()
 
             var update = issue.ToUpdate();
 
-            Assert.Null(update.Labels);
+            Assert.NotNull(update.Labels);
```

Comment:
Because in json String we add labels.
```
""labels"": [
{
    ""url"": ""https://api.github.com/repos/octocat/Hello-World/labels/bug"",
    ""name"": ""bug"",
    ""color"": ""f29513""
}
```
So this test will always fails if we test null.